### ISSUE TITLE
validate warns "document standard id ... is not correct" on good labels

### ISSUE THREAD

**Issue by rchenatjpl** [NASA-PDS-Incubator/validate#429]
## 🐛 Describe the bug
Validate gives erroneous WARNINGs "document standard id ... is not correct"

## 📜 To Reproduce
Unzip the attachment, then
% validate -t EPPS_EDR_SIS.xml
[snip]
  PASS: file:/Users/rchen/Desktop/EPPS_EDR_SIS.xml
      WARNING  [warning.file.not_msword_mimetype]   file:/Users/rchen/Desktop/EPPS_EDR_SIS.DOC with document standard id 'Microsoft Word' is not correct
      WARNING  [warning.file.not_text_mimetype]   file:/Users/rchen/Desktop/EPPS_EDR_SIS.TXT with document standard id '7-Bit ASCII Text' is not correct
        1 product validation(s) completed

## 🕵️ Expected behavior
No validation errors

## 🩺 Test Data / Additional context
[Archive.zip](https://github.com/NASA-PDS/validate/files/7417071/Archive.zip)

## 🖥 System Info

---

## 🦄 Related requirements
:unicorn: #367

## ⚙️ Engineering Details
@qchaupds a few things looks like they need to be updated here:
- [ ] we are not being case agnostic in our file suffix patterns (see JPEG and MSWORD, as examples). let's just ignore case.

- [ ] text file check needs to be simplified and may just not be working
```
$ validate-2.2.0-SNAPSHOT/bin/validate -t src/test/resources/github367/document/invalid_pccref.xml
  PASS: file:/Users/jpadams/Documents/proj/pds/pdsen/workspace/validate/src/test/resources/github367/document/invalid_pccref.xml
      WARNING  [warning.file.not_text_mimetype]   file:/Users/jpadams/Documents/proj/pds/pdsen/workspace/validate/src/test/resources/github367/document/pccref.txt with document standard id '7-Bit ASCII Text' is not correct
        1 product validation(s) completed

Summary:

  0 error(s)
  1 warning(s)

  Product Validation Summary:
    1          product(s) passed
    0          product(s) failed
    0          product(s) skipped

  Referential Integrity Check Summary:
    0          check(s) passed
    0          check(s) failed
    0          check(s) skipped

  Message Types:
    1            warning.file.not_text_mimetype

End of Report
Completed execution in 3650 ms
```
But why is that invalid? I even removed the file we had in the test resources and created my own and it says it is 
invalid [pccref.txt](https://github.com/NASA-PDS/validate/files/7419892/pccref.txt)?
```
$ file src/test/resources/github367/document/pccref.txt
src/test/resources/github367/document/pccref.txt: ASCII text
```
For text files, I think we should really simplify this check since there are really infinite number of file suffixes possible for text files (if they even have a file suffix). Let's just check that it is one of the more common image/document formats (check it does not match any of the other file suffixes, excluding HTML, Latex, Postcript, RTF since those are basically text so it doesn't really matter from an archive sense)

- [x] For the error messaging, can we change the wording to something like:
```
WARNING  [warning.file.not_msword_mimetype]   file:/Users/rchen/Desktop/EPPS_EDR_SIS.DOC with document standard 'Microsoft Word' is incorrect. Expected file suffix(es): [ doc, docx ]
```

- [x] For MSWord, we need to add additional file suffixes: docm, dot, dotm, dotx. For Excel we need to add: xltx, xlt, xlsm, xlsb

**Comment by tloubrieu-jpl** [comment]
Should be fixed by #419

**Comment by qchaupds** [comment]
Upon further reflection, this issue will need more work to get it correct.

**Comment by jordanpadams** [comment]
@qchaupds https://github.com/NASA-PDS/validate/pull/428 did not solve this issue. please take a look when you can.

**Comment by rchenatjpl** [comment]
Debra Kazden and Joe Mafi of PPI just hit this bug with a 7-Bit ASCII Text file

**Comment by jstone-psi** [comment]
This is happening to SBN with UTF-8 text files, as well.
